src/UI/styles/autofill-tooltip-styles.css

```diff
@@ -291,6 +292,7 @@ hr:first-child {
     border: 0;
     cursor: pointer;
     display: inline-block;
+    font-family: inherit;
```

Comment:
These properties are inherited even in the shadow DOM unless explicitly set. Fixed now.